airtime_mvc/public/js/airtime/schedule/full-calendar-functions.js

```diff
@@ -214,7 +214,13 @@ function eventRender(event, element, view) {
                     $(element)
                     .find(".fc-event-time")
                     .before('<span class="small-icon linked"></span><span class="small-icon show-empty"></span>');
-                } else {
+                   // in theory a linked show shouldn't have an automatic playlist so adding this here
```

Comment:
seems sensible.

Reply:
So I put in most of the changes you requested. I didn't modify the JS code because it was working as intended, the comment was just pointing out the lack of a need for another branch where linked would also have an autoplaylist, but the two other branches were necessary.

Also the image is working locally but I'm not sure if there is a problem with git and binary files again. Ever since I merged some document change commits I have started running into issues like this 
warning: CRLF will be replaced by LF in airtime_mvc/public/css/images/icon_alert_cal_autoplaylist.png.

If there is an issue with the png then this is likely the case. 

Although it is basically seeing binary png and jpgs that have been updated as modified on my local git directory as we saw in #202 and I've needed to do my hard reset command referenced there to be able to do anything with my local repo. I tried updating git to via PPA to see if that makes a difference.

Reply:
I think the image issue might have something to do with the `.gitattributes` file. I'll have a look at the image locally and merge if it looks ok there.

Reply:
I just re-checked but somehow the file really does seem corrupted.

Here is what pngcheck is reporting:

```
pngcheck -v airtime_mvc/public/css/images/icon_alert_cal_autoplaylist.png
File: airtime_mvc/public/css/images/icon_alert_cal_autoplaylist.png (571 bytes)
  File is CORRUPTED.  It seems to have suffered DOS->Unix conversion.
ERRORS DETECTED in airtime_mvc/public/css/images/icon_alert_cal_autoplaylist.png
```

Reply:
Ok, I'll push a commit with the original file and make sure it doesn't get corrupted shortly.